src/store/getters/state-getters.ts

```diff
@@ -1072,3 +1078,15 @@ export const hideChangeButtonForSoleProps = (state: StateIF, getters): boolean =
   const isDba = isProprietor && isOrganization
   return !getters.isRoleStaff && isDba
 }
+
+export const getExpiryDateString = (state: StateIF): string => {
+  return state.stateModel.restoration?.expiry
+}
+
+export const getFormattedExpiryText = (state, getters): string => {
+  if (getters.getExpiryDateString) {
+    return DateUtilities.monthDiffToToday(getters.getExpiryDateString) + ' months, expires on ' +
+      DateUtilities.yyyyMmDdToPacificDate(getters.getExpiryDateString)
+  }
+  return '[no expiry date]'
+}
```

Comment:
I'd like an opportunity to refactor this and create a module for Restorations.  These getters, and others, would then be moved to the restoration module's getters.

Reply:
If it fits within the estimate for this ticket, go for it.

(An estimate of 3 should equal 1-3 days of work.)

Reply:
Was a decision made re: this conversation?

Reply:
With the time remaining in the sprint and the other tickets assigned to me, I don't think I have time to dig into this at present.